packages/sdk/src/machines/purchase-initiation-machine/index.ts

```diff
@@ -389,6 +395,35 @@ export const purchaseInitiationMachine = setup({
     ) => {
       return !isERC20AllowanceSufficient(context.project, allowance);
     },
+    isSaleStarted: ({ context }) => {
+      const project = context.project;
+      const liveSaleData: LiveSaleData =
+        context.projectSaleManagerMachine.getSnapshot().context.liveSaleData;
+      const walletClient = context.artblocksClient.getWalletClient();
+
+      if (!liveSaleData || !walletClient?.account) {
```

Comment:
Not 100% sure how `isSaleStarted` is used (but I am reading it as "has the sale for the project started already") in which case couldn't the sale be started even if the users wallet isn't connected via `walletClient?.account`? (but maybe that isn't the intention of this and its more of a "has the sale started and can I buy it"?)

Reply:
Ah yeah it's more meant to be "is the project purchasable by the user" will update the name to make more clear!